src/bank.rs

```diff
@@ -103,6 +104,7 @@ struct ErrorCounters {
     account_not_found_validator: usize,
     account_not_found_leader: usize,
     account_in_use: usize,
+    old_last_id: usize,
```

Comment:
Define `old`. Maybe this should be `last_id_age` or `entry_id_age`?

Reply:
expired_last_id?

Reply:
last_id_not_found?